addons/test_mail/tests/test_performance.py

```diff
@@ -697,6 +697,116 @@ def test_complex_tracking_template(self):
         self.assertEqual(rec1.message_ids[2].notified_partner_ids, self.partners | self.user_portal.partner_id)
         self.assertEqual(len(rec1.message_ids), 3)
 
+    @mute_logger('odoo.tests', 'odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('__system__', 'emp')
```

Comment:
What is the deciding factor?

Reply:
We are talking of testing for emp only for a while, usually emp has more query than admin so more coverage, and this method will usually be called by a user I think, not the system. The main goal is to avoid to add useless processing/test time.

Reply:
Ok, either way is fine for me, I'll keep just `emp` then